### PR TITLE
Add linters on azure

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 license_file = LICENSE
 
 
+[tool:pytest]
+addopts = --cov=yarl -v
+filterwarnings=error
+norecursedirs = dist docs build .tox .eggs venv virtualenv .git
+minversion = 3.8.2
+testpaths = tests/
+junit_suite_name = yarl_test_suite
```
Comment (webknjaz) [inline setup.cfg]: @tonybaloney released https://pypi.org/project/pytest-nunit/ in recent months. AFAIU this format supports richer reporting. @asvetlov do you want to give it a try?

File: setup.cfg [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-
 [metadata]
 license_file = LICENSE
 
 
+[tool:pytest]
+addopts = --cov=yarl -v
+filterwarnings=error
+norecursedirs = dist docs build .tox .eggs venv virtualenv .git
+minversion = 3.8.2
```
Comment (webknjaz) [inline setup.cfg]: how about 4.0/5.0?

File: setup.cfg (hunk 1 above)
Comment (asvetlov) [reply setup.cfg]: Thanks for the link.
Created https://github.com/aio-libs/yarl/issues/355

Now I want to make everything working, left everything not required to further PRs

File: setup.cfg (hunk 2 above)
Comment (asvetlov) [reply setup.cfg]: This is a copy-paste from other project.
Please feel free to upgrade to newer versions.

**Pull request merged**